js/helpers.js

```diff
@@ -1,16 +1,5 @@
 import Adapt from 'core/js/adapt';
-
-const defaultAriaLevels = {
-  '_menu': 1,
-  '_menuGroup': 2,
-  '_menuItem': 2,
-  '_page': 1,
-  '_article': 2,
-  '_block': 3,
-  '_component': 4,
-  '_componentItem': 5,
-  '_notify': 1
-};
+import a11y from './a11y';
```

Comment:
should a11y be capitalised to A11y?

Reply:
no, it's an instance at this point, not a class.